docs/detections/prebuilt-rules/prebuilt-rules-changelog.asciidoc

```diff
@@ -1,15 +1,17 @@
 [[prebuilt-rules-changelog]]
 == Prebuilt rule changes per release
 
-[float]
-=== 7.12.0
+beta[]
```

Comment:
@brokensound77 should we remove the `[beta]` warning? I thought we were out of beta, but can't remember for sure.

Reply:
should this change happen in 7.12.1 docs?

Reply:
@brokensound77 yes, this went GA in 7.11 so we can remove the `beta` tag.

Reply:
as opposed to 7.12.0